modules/controller_pool/controller-primary.tpl

```diff
@@ -2,10 +2,37 @@
 
 export HOME=/root
```

Comment:
nitpic:

We should probably use $HOME instead of "/root" throughout this script, since we have gone through the effort of defining the variable.

IIRC, this was a fix for Docker which requires the variable be set, and we are assured that /root exists based on the operating systems that this module will deploy.